packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx

```diff
@@ -201,12 +186,6 @@ export const RoleSelect = (props: {
             as='span'
             gap={1}
           >
-            <Text
-              as='span'
-              sx={t => ({ color: t.colors.$blackAlpha400 })}
-            >
-              {`${t(localizationKeys('formFieldLabel__role'))}:`}
-            </Text>
```

Comment:
Why is the `Role:` word removed here?

Reply:
As shown in the designs, the Dropdown for the role inside the `<MemberListTable/>`, should not contain he `Role:`. The `Role:` text is only needed when the `<RoleSelect/>` is inside the `<InviteMembersForm/>`, I will need to change that as we use `<RoleSelect/>` in both places.